src/test/java/org/opensearch/ad/transport/handler/AbstractIndexHandlerTest.java

```diff
@@ -73,6 +74,9 @@ enum IndexCreation {
     @Mock
     protected IndexNameExpressionResolver indexNameResolver;
 
+    @Mock
+    protected OpenSearchAsyncClient javaAsyncClient;
```

Comment:
Can you mock the client in `setUp` like:

`sdkJavaAsyncClient = mock(OpenSearchAsyncClient.class);`

Reply:
ACk